react/hooks/usePosition.ts

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from 'react'
+
+interface Coordinate {
+  latitude: number | undefined
+  longitude: number | undefined
+}
+
+interface Position {
```

Comment:
Maybe something like `LatLongRecord`, `LatLongHistory` or `LatLongTime` is a better fit here